src/backend/test/unit/common/helpers/pdf-helper.spec.ts

```diff
@@ -0,0 +1,42 @@
+import * as pug from 'pug';
+import * as Puppeteer from 'puppeteer';
+import { pugToPdfBuffer } from 'src/common/helpers/pdf-helper';
+/**
+ * @Description
+ * This file tests the contents of common/helpers/pdf-helper.ts
+ */
+describe('PdfHelper', () => {
+  /**
+   * @method pugToPdfBuffer
+   *
+   * @description
+   * This test suite validates that the method does not throw error and returns the expected data
```

Comment:
Again, would add the general input that we are testing against and expected output